### PR TITLE
Frontend doesn't show demo result

### PR DESCRIPTION
**Pull request by sanyavertolet**
### What's done:
 * Fixed frontend error when no demo result was shown
 * Fixed deserialization error on `demoView`
 * Fixed logging in `KubernetesService`

### CODE REVIEW

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/DemoRunComponent.kt [hunk 1]
```diff
@@ -42,6 +43,8 @@ private val defaultCode = """
     |
     |fun main() {
     |    val SAVEOUR = "tool"
+    |
```
Comment (nulls) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/DemoRunComponent.kt:46]: do we need it? Or is it to have a formatting issue in stub example?

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/demo/DemoView.kt [hunk 2]
```diff
@@ -111,3 +89,23 @@ private fun demoView(): FC<DemoViewProps> = FC { props ->
         }
     }
 }
+
+/**
+ * [Props] for DemoView
+ */
+external interface DemoViewProps : Props {
```
Comment (nulls) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/demo/DemoView.kt:96]: suggestion to add a flag to disable custom-selector

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/demo/DemoView.kt [hunk 3]
```diff
@@ -53,35 +51,14 @@ val demoView: VFC = VFC {
...
-    var demoRunEndpoint: String
-
-    /**
-     * Optional config name for this demo
-     */
-    var configName: String?
-}
-
-private fun demoView(): FC<DemoViewProps> = FC { props ->
+private val commonDemoView: FC<DemoViewProps> = FC { props ->
```
Comment (nulls) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/demo/DemoView.kt:61]: so, `val` is better? not a `fun`?

**Review by nulls — APPROVED** [review]

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/DemoRunComponent.kt (hunk 1 above)
Comment (sanyavertolet) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/DemoRunComponent.kt:46]: Yep, it is to make `ktlint` find some formatting issues

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/demo/DemoView.kt (hunk 3 above)
Comment (sanyavertolet) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/demo/DemoView.kt:61]: making `FC` be `val` allows to remove line 
```
val commonDemoView = demoView()
```

Actually there is no other difference

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/demo/DemoView.kt (hunk 2 above)
Comment (sanyavertolet) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/demo/DemoView.kt:96]: Should be done in PR with several run modes (if you are talking about disabling demoMode selector). Probably it is the task that should be done next

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/demo/DemoView.kt (hunk 3 above)
Comment (nulls) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/demo/DemoView.kt:61]: > Actually there is no other difference

So it's only refactoring, isn't it?

Comment (sanyavertolet) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/demo/DemoView.kt:61]: We used to create a new demoView instance, which was recreated during each re-render (and that caused no result on frontend despite the fact that the result was received). Now we use commonDemoView that is created once.

Comment (nulls) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/demo/DemoView.kt:61]: so `val` is better :)

**Pull request merged**